UI/src/components/molecules/RipaFormStep6.vue

```diff
@@ -1,6 +1,20 @@
 <template>
   <v-form ref="stepForm" lazy-validation>
-    {{ model.agencyQuestions }}
+    <div>model.agencyQuestions: {{ model.agencyQuestions }}</div>
+    <div>agencyQuestions: {{ agencyQuestions }}</div>
+
+    <template v-if="questions.length > 0">
```

Comment:
With switching to RipaAgencyQuestions (vs. a single question), we can move the v-for to the questions component and just pass the stop model through v-model.

`
<template>
  <v-form ref="stepForm" lazy-validation>
    {{ model.agencyQuestions }}

    <ripa-agency-questions
      v-model="model"
      :on-open-statute="onOpenStatute"
    ></ripa-agency-questions>

    <v-spacer></v-spacer>
`